docs/blog/2025-09-langflow-poc.md

```diff
@@ -0,0 +1,53 @@
+---
+date: 2025-09-12
+title: "Langflow on Stepflow POC"
+description: "Demonstrates the use of Stepflow to execute Langflow flows."
+slug: langflow-poc
+authors:
+  - benchambers
+tags: [announcements]
+draft: true
+---
+
+# Langflow on Stepflow POC
+
+Stepflow was designed to provide a common runtime for GenAI workflow frameworks.
+We've been working on a proof-of-concept converting Langflow flows Stepflow and executing them.
+This post demonstrates the integration and discusse some of the steps that were necessary.
+
+<!-- truncate -->
+
+## Demo
+
+For this demo, we created a Langflow flow using the Blog Content generation template.
+We configured it to fetch content from `stepflow.org/docs`, and saved it as [`flow.json`](/files/2025-09-langflow-poc-flow.json).
+
+![Blog Content Flow](/img/2025-09-langflow-poc-flow.png)
+
+The following command converts the Langflow file to Stepflow and executes it.
+You should set `OPENAI_API_KEY` following environment variables (or manually replace it).
```

Comment:
Sentence is weird - 'You will need to set the OPEN_API_KEY environment variable (or manually replace it). 

Also, use this as the place to make the first mention of overlaying with an extra sentence or two.